src/openms/include/OpenMS/FORMAT/MRMFeatureQCFile.h

```diff
@@ -127,6 +131,69 @@ namespace OpenMS
     std::map<String, std::pair<double,double>>& meta_values_qc
   ) const;
 
+  /**
+    @brief Extracts a column's value from a line.
```

Comment:
I like the encapsulation of this into a seperate method with overloads for different values.